pkg/controllers/vdb/sandboxsubcluster_reconciler.go

```diff
@@ -204,6 +205,11 @@ func (s *SandboxSubclusterReconciler) executeSandboxCommand(ctx context.Context,
 			if err != nil {
 				return ctrl.Result{}, err
 			}
+
+			res, err = s.waitForSandboxScUp(ctx, sb, sc)
+			if err != nil {
+				return res, err
```

Comment:
We should just log the error and not end this method so other subclusters can be added to the sandbox without a reiteration.

Reply:
if this is the first subcluster to be sandboxed we can't continue until it's up, since we need to tell both main and sandbox about new subclusters being added.

Reply:
If we poll subclsuter up failed, we will never come here again since the subcluster is correct in vdb.Status so next retry will skip that subcluster. We need to call this function in an earlier stage.